layout/base/nsRefreshDriver.cpp

```diff
@@ -1000,20 +1000,29 @@ void nsRefreshDriver::CreateVsyncRefreshTimer() {
     return;
   }
 
+  // Diagnostic for https://github.com/RecordReplay/backend/issues/5734
+  recordreplay::RecordReplayAssert("nsRefreshDriver #10 mOwnTimer=%s sRegularRateTimer=%s isParentProcess=%s",
+    !!mOwnTimer ? "yes" : "no",
+    !!sRegularRateTimer ? "yes" : "no",
+    XRE_IsParentProcess() ? "yes" : "no");
```

Comment:
XRE_IsParentProcess() will never return true in a recording process, we don't need to include that in the assert.

Reply:
Figured that was the case since we record content processes only, but as troubled mathematician Max Cohen says in Darren Aronofski's cult hit movie "Pi", _'...restate my assumptions...'_.

I'll get rid of it :)